themes/10up-theme/includes/core.php

```diff
@@ -60,6 +60,72 @@ function theme_setup() {
 		)
 	);
 
+	add_theme_support(
+		'editor-font-sizes',
+		array(
+			array(
+				'name'      => __( 'Extra Extra Small', 'tenup-theme' ),
```

Comment:
@AlejandroPerezMartin Can we use 2x small or something like that to avoid "extra extra"?

Reply:
sure, renamed!